static/js/controllers/appcontroller.js

```diff
@@ -58,6 +58,7 @@ define(["jquery", "angular", "underscore"], function($, angular, _) {
 				}
 			}
 		};
+		$scope.defaults.settings.sounds = playSound.getDefaults();
```

Comment:
This does not work, as it does not clone the defaults. It stores a reference to the defaults. Whenever you change the stuff in sounds you change the defaults.